treerec/tskit/tree_sequence.c

```diff
@@ -0,0 +1,2438 @@
+/*
+** Copyright (C) 2015-2018 University of Oxford
+**
+** This file is part of msprime.
+**
+** msprime is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 3 of the License, or
+** (at your option) any later version.
+**
+** msprime is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with msprime.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include <stdbool.h>
+
+#include "gsl_math.h"
```

Comment:
@bhaller I see you've switched out MSP_MIN/MAX for GSL_MIN/MAX; just wondering why?

Reply:
I don't think I did this...?

Reply:
Oh, I see - the change was on the other side. =)